File: WEB/HOSPITEC/src/app/models/paciente/paciente.ts
export class Paciente {
  cedula: string;
  nombre: string;
  apellidos: string;
  telefono: string;
  nacimiento: string;
  direccion: string;
  tratamiento: string;
  patologias: string;
  doctorencargado: string;
  estado: string;
  contraseña: string;



  constructor(ced, nom, ape, tel, fecha, dir, trat, pat,dr,est,cont) {
    this.cedula = ced;
    this.nombre = nom;
    this.apellidos = ape;
    this.telefono = tel;
    this.nacimiento = fecha;
    this.direccion = dir;
    this.tratamiento = trat;
    this.patologias = pat;
    this.doctorencargado = dr;
    this.estado = est;
    this.contraseña = cont;
  }

}

File: Hospital.Api/Hospital.Api/Models/paciente.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;
using NodaTime;

namespace Hospital.Api.Models
{
    public class paciente
    {
        [Key]
        public string cedula { get; set; }
        public string nombre { get; set; }
        public string apellidos { get; set; }
        public string telefono { get; set; }
        public string nacimiento { get; set; }
        public string direccion { get; set; }
        public string patologias { get; set; }
        public string tratamiento { get; set; }
        public string doctorencargado { get; set; }
        public string estado { get; set; }
        public string contraseña { get; set; }


    }
}
